integrations/taskmarket/requester.py

```
@@ -0,0 +1,87 @@
+import json, subprocess, shutil, sys
+
+BASE_CHAIN_ID = 8453  # Base mainnet
+SAFE_COMMANDS = ("init", "wallet", "address", "task", "inbox", "agents", "stats")
+
+class TaskMarketError(Exception):
+    pass
+
+def _run(args, authorize=False):
+    if not shutil.which("npx"):
+        raise TaskMarketError("npx/taskmarket CLI not found on PATH")
+    cmd = ["npx", "taskmarket"] + args
```

Ruff (0.16.1)

[warning] Consider `["npx", "taskmarket", *args]` instead of concatenation

Replace with `["npx", "taskmarket", *args]`

(RUF005)